app/javascript/packs/runner_application.js

```diff
@@ -64,4 +64,7 @@ if(autocompleteComponent) {
 
 // Little bit hacky but we want to prevent the
 // Cookie banner from showing in preview mode.
-document.getElementById("govuk-cookie-banner").style.display = "none";
+const cookieBanner = document.getElementById("govuk-cookie-banner");
+if(cookieBanner) {
+  cookieBanner.style.display = "none";
+}
```

Comment:
This change is not directly related, but I was getting a console error locally due `document.getElementById('govuk-cookie-banner')` returning nil, so I wrapped it to ensure the eleemnt exists before we try to style it.